### DISCUSSION TITLE
fix the RSS function in K2

### DISCUSSION THREAD

**Discussion by litho-publisher** [eworkers/K2ForJ4#78] [General]
hello
I would love it if someone could fix the RSS function in K2.
Unfortunately since some versions only the pages are downloaded.
I would pay for that too.
Please reply as soon as possible.
Thank you very much
Litho

**Comment by eworkers** [comment]
> hello I would love it if someone could fix the RSS function in K2. Unfortunately since some versions only the pages are downloaded. I would pay for that too. Please reply as soon as possible. Thank you very much Litho

Hello, No need to pay for anything
Just describe your problem properly, if it is a K2ForJ4 issue we will fix it and we'll be thankful that you have reported it

Can you at least provide an active rss link that is being downloaded as you're describing?

**Comment by litho-publisher** [comment]
HI thank you very much for your quick reply and your offer.
The error with the RSS has existed for a long time.
If you activate the rrs button in the categories -> Display settings -> option categories "Atom/RSS feed link (in the document head)" and "Atom/RSS feed icon" then the RSS logo is displayed in the frontend, but if you click on it, an rss feed is not displayed, but the current page is downloaded.
Hope that will be easy to fix.
Regards
Litho

**Reply by eworkers** [reply]
Please share at least one link where the problem exist

**Reply by litho-publisher** [reply]
Here an example
https://www.snookermania.de/national.html
push on the RSS sign and will get an download not an rss feed.

**Reply by eworkers** [reply]
your rss is just fine
keep in mind that rss are meant to be consumed by rss readers and to be displayed as regular web page, however the fact that the feed is being downloaded in your case is not related to K2ForJ4, it might be related to your browser

didn't download in my case and the response looks very normal, below is a screenshot for the header response and everything is just normal, your users that wishes to consume your K2 rss feeds are not having any problems

![header_res](https://github.com/eworkers/K2ForJ4/assets/115772544/4ec1dbf9-508e-4dde-b13a-6e2b3fb38da9)

**Comment by krmrtrzv** [comment]
I wonder if RSS feeds are still in the loop. Lots of people consider it a
dead instrument

**Comment by litho-publisher** [comment]
Well it's the easiest way to push news to social media with RSS and ifttt.com, fully automated. I haven't found another tool for this yet... Suggestions welcome.
